docs/build/get-started/get-started.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_label: Get started
+sidebar_label: Overview
```

Comment:
It looks like some of the content on the Get started page is repeated on this page? What did you have in mind? 😀 Maybe we don't need the Overview page? WDYT?

Reply:
You are right! the get started page was meant to be removed.